src/styles/Editor.scss

```diff
@@ -42,9 +42,9 @@
 }
 
 .editor-expand-panel-arrow-hidden {
-  padding: 1rem;
-  flex: 0 1 auto;
-  width: 46px;
+  // padding: 1rem;
+  // flex: 0 1 auto;
+  // width: 46px;
 }
```

Comment:
In the future, we'll probably remove this entire component and/or figure out a more elegant solution. For now, I'm fine with this change.